v3/docs/gs/params.md

```diff
@@ -32,7 +32,12 @@ The following global parameters are part of the stable API.
 | `logging.path` | `$PWD/hse.log` | Log file when logging.destination==file |
 | `logging.level` | `7` | Logging severity level (0==emergency; 7==debug) |
 | `socket.enabled` | `true` | REST interface mode (false==disabled, true==enabled) |
-| `socket.path` | `/tmp/hse-<pid>.sock` | UNIX domain socket file when socket.enabled==true |
+| `socket.path` | `$XDG_RUNTIME_DIR/hse-<pid>.sock` | UNIX domain socket file when socket.enabled==true |
+
+!!! note
+    If `$XDG_RUNTIME_DIR` is not defined then the default value for `socket.path`
+    is `/tmp/hse-<pid>.sock`.
+
```

Comment:
The socket path params have been renamed to `rest.enabled` and `rest.socket_path`.